src/System.Linq.Expressions/src/Resources/Strings.resx

```diff
@@ -156,6 +156,9 @@
   <data name="SetterMustBeVoid" xml:space="preserve">
     <value>Setter should have void type.</value>
   </data>
+  <data name="PropertyTypeMustMatchGetter" xml:space="preserve">
+    <value>Property type must match the value type of Getter</value>
```

Comment:
nit: uppercase `G` is inconsistent with lower case `s` in `PropertyTypeMustMatchSetter`